src/main/java/mate/academy/rickandmorty/service/impl/CharacterServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.rickandmorty.service.impl;
+
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterResponseDto;
+import mate.academy.rickandmorty.exception.NoSuchCharacterException;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CharacterServiceImpl implements CharacterService {
+    private final CharacterRepository repository;
+    private final CharacterMapper mapper;
+    private final Random random = new Random();
```

Comment:
declare random as bean and autowire it
```suggestion
    private final Random random;
```